.buildkite/pipeline.yml

```diff
@@ -25,7 +25,7 @@ steps:
     <<: *docker-container
     command: |
       cp gradle.properties-example gradle.properties
-      ./gradlew lintWordpressVanillaRelease
+      ./gradlew lintWordpressVanillaRelease --info
```

Comment:
@zwarm I see you ended up merging this debugging change after all. Probably not a big deal overall, but since we try to avoid too much verbosity on BuildKite to avoid logs for the common, working case being less readable and being truncated for length, we might want to revert this?

(cc @jkmassel just FYI – in case you saw that flag in `develop` later and would wonder why it got there)

Reply:
I just saw this while reviewing another PR. @zwarm Unless you really need the debug logs in the CI all the time, I'd like to open a PR to remove this. This makes the logs very verbose and makes it a lot harder to find the important ones. In my experience, flags like `--info`, `--stacktrace` etc are useful to debug how Gradle is handling something specific and very rarely useful to debug the Gradle task and its result.

Reply:
@oguzkocer - I agree with you, please go ahead and push the PR. Thank you.

Reply:
Thanks @zwarm 👍